crates/sui-core/src/unit_tests/data/transitive_dependencies/package_root_depends_on_A/sources/trusted_coin.move

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/// Example coin with a trusted owner responsible for minting/burning (e.g., a stablecoin)
+module examples::trusted_coin {
```

Comment:
Note that Root does not actually depend on A -- i.e. the code is not written in such a way that it would reference A so hypothetically, while we can't build Root without having A's source, we could publish it.